views/menus/show.pug

```diff
@@ -3,15 +3,17 @@ extends ../application.pug
 block content
 
     .col-md-4.offset-md-3
-      img(src= menu.image + '.jpg' width="450px" height="300px")
+      img(src= menu.image width="450px" height="300px")
     .col-md-2
       .row
         h2= menu.title
         hr
       .row
-        img(src="/img/heart.svg.png" width="200px" height="200px")
+        form(action="/menus/" + menu.id + "/votes" method="POST")
+          input(type="hidden" name="MenuId" value=menu.id)
```

Comment:
you don't need this hidden field if you are making the request to /menus/:menu_id/votes